src/app/components/normal-component/pick-game/pick-game.component.ts

```diff
@@ -138,6 +141,8 @@ class GameDescription {
 
     public static readonly KAMISADO: Localized = () => $localize`Your goal is simple: reach the last line. But the piece you move depends on your opponent's last move!`;
 
+    public static readonly LASCA: Localized = () => $localize`Similar to checkers, capture opponent pieces, free your own, and immobilize your opponent to win the game!`;
```

Comment:
je dirais plutôt "capture pieces of the opponent" non ? Là ça sonne un peu "capture pièce opposant !!"

Reply:
capture opponent's pieces ?

Reply:
tapafééé

Reply:
Bah si !